varfish/static/js/filter_form.js

```diff
@@ -1280,6 +1280,179 @@ let presets_genelist = {
       "MEN1",
       "RB1"
     ]
+  },
+  "acmg30": {
+    "field": "gene_allowlist",
+    "data": [
+      "ACTA2",
+      "ACTC1",
+      "ACVRL1",
+      "APC",
+      "APOB",
+      "ATP7B",
+      "BAG3",
+      "BMPR1A",
+      "BRCA1",
+      "BRCA2",
+      "BTD",
+      "CACNA1S",
+      "CASQ2",
+      "COL3A1",
+      "DES",
+      "DSC2",
+      "DSG2",
+      "DSP",
+      "ENG",
+      "FBN1",
+      "FLNC",
+      "GAA",
+      "GLA",
+      "HFE",
+      "HNF1A",
+      "KCNH2",
+      "KCNQ1",
+      "LDLR",
+      "LMNA",
+      "MAX",
+      "MEN1",
+      "MLH1",
+      "MSH2",
+      "MSH6",
+      "MUTYH",
+      "MYBPC3",
+      "MYH11",
+      "MYH7",
+      "MYL2",
+      "MYL3",
+      "NF2",
+      "OTC",
+      "PALB2",
+      "PCSK9",
+      "PKP2",
+      "PMS2",
+      "PRKAG2",
+      "PTEN",
+      "RB1",
+      "RBM20",
+      "RET",
+      "RPE65",
+      "RYR1",
+      "RYR2",
+      "SCN5A",
+      "SDHAF2",
+      "SDHB",
+      "SDHC",
+      "SDHD",
+      "SMAD3",
+      "SMAD4",
+      "STK11",
+      "TGFBR1",
+      "TGFBR2",
+      "TMEM127",
+      "TMEM43",
+      "TNNC1",
+      "TNNI3",
+      "TNNT2",
+      "TP53",
+      "TPM1",
+      "TRDN",
+      "TSC1",
+      "TSC2",
+      "TTN",
+      "TTR",
+      "VHL",
+      "WT1",
+    ]
+  },
+  "acmg31": {
```

Comment:
This actually misses the genes from v3.1 (BAG3, DES, RBM20, TNNC1, and TTR). Also some duplicates are contained here (see below)